dev-docs/source/MantidUsedIconsTable.rst

```diff
@@ -0,0 +1,130 @@
+.. _MantidUsedIconsTable:
+
+.. |copy| image:: images/LocalIcons/content-copy.png
+.. |paste| image:: images/LocalIcons/content-paste.png
+.. |cut| image:: images/LocalIcons/content-cut.png
+.. |settings| image:: images/LocalIcons/settings.png
+.. |eraser| image:: images/LocalIcons/eraser.png
+.. |delete| image:: images/LocalIcons/trash-can.png
+.. |error| image:: images/LocalIcons/asterisk.png
+.. |plot| image:: images/LocalIcons/chart-line.png
+.. |plot-multiple| image:: images/LocalIcons/chart-areaspline.png
+.. |up| image:: images/LocalIcons/arrow-up.png
+.. |down| image:: images/LocalIcons/arrow-down.png
+.. |play| image:: images/LocalIcons/play.png
+.. |stop| image:: images/LocalIcons/square.png
+.. |pause| image:: images/LocalIcons/pause.png
+.. |plus| image:: images/LocalIcons/plus.png
+.. |close| image:: images/LocalIcons/close.png
+.. |search| image:: images/LocalIcons/folder.png
+.. |refresh| image:: images/LocalIcons/twitter-retweet.png
+.. |home| image:: images/LocalIcons/home.png
+.. |pan| image:: images/LocalIcons/arrow-all.png
+.. |zoom| image:: images/LocalIcons/magnify-plus-outline.png
+.. |grid| image:: images/LocalIcons/grid.png
+.. |save| image:: images/LocalIcons/content-save.png
+.. |transfer-to-table| image:: images/LocalIcons/file-move.png
+.. |beam-centre| image:: images/LocalIcons/adjust.png
+.. |diagnostic| image:: images/LocalIcons/stethoscope.png
+.. |process-all| image:: images/LocalIcons/sigma.png
+
+.. |table| image:: images/LocalIcons/table.png
+.. |export-data| image:: images/LocalIcons/file-export.png
+.. |add-row| image:: images/LocalIcons/table-row-plus-after.png
+.. |remove-row| image:: images/LocalIcons/table-row-remove.png
+.. |new-table| image:: images/LocalIcons/table-plus.png
+.. |remove-table| image:: images/LocalIcons/table-remove.png
+.. |expand| image:: images/LocalIcons/expand-all.png
+.. |collapse| image:: images/LocalIcons/collapse-all.png
+
+Mantid Icon Table
+#################
+
+Here is a repository of all currently used icons in Mantid.
+These icons are from Material Design Icons an open source
+icon project by google. Best place to find new unused icons
+is on the `cheatsheet. <https://cdn.materialdesignicons.com/3.6.95/>`_
+
+General Icons
+-------------
+
++---------------+----------------------------+---------------------+
+| Use           | getIcon String             | Icon                |
++===============+============================+=====================+
+| Copy          | "mdi.content-copy"         | |copy|              |
++---------------+----------------------------+---------------------+
+| Paste         | "mdi.content-paste"        | |paste|             |
++---------------+----------------------------+---------------------+
+| Cut           | "mdi.content-cut"          | |cut|               |
++---------------+----------------------------+---------------------+
+| Settings      | "mdi.settings"             | |settings|          |
```

Comment:
This has been changed in ISIS SANS to `settings-outline`. The filled-in `settings` icon is predominantly black, so fits in less well with the other icons